official/utils/flags/_base.py

```diff
@@ -133,3 +139,9 @@ def get_num_gpus(flags_obj):
   from tensorflow.python.client import device_lib  # pylint: disable=g-import-not-at-top
   local_device_protos = device_lib.list_local_devices()
   return sum([1 for d in local_device_protos if d.device_type == "GPU"])
+
+def apply_clean(flags_obj):
```

Comment:
I think it is not ideal to put this function in the flag module since it has nothing to do with flag itself.

Reply:
Sure. Moved to `model_helpers`